src/styles/blocks/card.scss

```diff
@@ -0,0 +1,106 @@
+@use 'variables' as v;
+
+.card {
+  box-sizing: border-box;
+  width: 200px;
+
+  padding: 32px 16px 16px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  transition: transform 300ms;
+
+  &:hover {
+    transform: scale(1.2);
+
+    &__title {
+      color: #34568b;
+    }
```

Comment:
The nested '&__title' selector inside ':hover' might not work as intended because it assumes '.card__title' is a child of '.card:hover'. Ensure that the HTML structure supports this or adjust the selector accordingly.